index.js

```diff
@@ -71,18 +75,35 @@ instance.prototype.incomingData = function(data) {
 		self.login = false;
 		self.status(self.STATUS_WARNING,'Checking Connection');
 		self.socket.write("2I"+ "\n"); // should respond with model description eg: "Streaming Media Processor"
+		self.socket.write("36I"+ "\n");

```

Comment:
self.socket.write("36I"+ "\n");
This dosn't need to be here, the response from 2I will trigger data.match and run it.

Reply:
Noted and removed.